src/Panel/Panel.php

```diff
@@ -137,6 +138,16 @@ public static function firewall(
 		return true;
 	}
 
+	/**
+	 * @since 4.3.0
+	 */
+	protected static function garbage(): void
+	{
+		// run garbage collection with a chance of 20%;
+		if (mt_rand(1, 10000) <= 0.2 * 10000) {
```

Comment:
Is 20% probability a good value for the Panel garbage collection?

Reply:
Maybe a bit too much. Using the Panel sends a lot of requests. I'd say we should aim for in average one execution for each short editing session (like open the Panel, navigate to a page, fix a typo). Based on that, I think 10% is probably more than enough.